doc/developers/contributing.rst

```diff
@@ -1112,3 +1112,32 @@ that implement common linear model patterns.
 
 The :mod:`sklearn.utils.multiclass` module contains useful functions
 for working with multiclass and multilabel problems.
+
+Estimator Tags
+--------------
+Scikit-learn introduced estimator tags in version 0.19.
+These are annotations of estimators that allow programmatic inspection of their
+capabilities, such as sparse matrix support, supported output types and
+supported methods.
+The estimator tags are a dictionary returned by the method ``_get_tags()``.
+These tags are used by the common tests and the ``check_estimator`` function to decide
+what tests to run and what input data is appropriate.
+
+The current set of estimator tags are:
+
+input_validation - whether the estimator does input-validation. This is only meant for stateless and dummy transformers!
```

Comment:
I don't get it, what does this mean? Only stateless and dummy transformers should set this to True? or to False? and why?

Might be clearer to document the default tag values here too